classes/State.py

```diff
@@ -92,30 +83,42 @@ def calculate_distance_matrix(self):
             distance_matrix.append(neighbour_distance)
         return distance_matrix
 
-    def get_max_number_of_swaps(self, cluster: Cluster):
+    @staticmethod
+    def get_max_number_of_swaps(vehicle: Vehicle):
         return min(
-            min(len(cluster.scooters), self.vehicle.battery_inventory),
-            len(self.current_location.get_swappable_scooters()),
+            min(len(vehicle.current_location.scooters), vehicle.battery_inventory),
+            len(vehicle.current_location.get_swappable_scooters()),
         )
```

Comment:
Isn't it natural to put this method into Vehicle?

Reply:
I agree!